plugins/spi/SPIDevice.h

```diff
@@ -86,7 +86,7 @@ class SPIDevice: public ola::Device {
   static const char SPI_DEVICE_NAME[];
   static const char HARDWARE_BACKEND[];
   static const char SOFTWARE_BACKEND[];
-  static const uint8_t MAX_GPIO_PIN = 25;
+  static const uint16_t MAX_GPIO_PIN = 25;
```

Comment:
That's what the code seems to do.

@nomis52 what are your thoughts, I'm inclined to say if the range is so large it becomes a bit of a pointless check. We could either remove it, or test against a series of offsets (0, 415, 1016), with each having a range up to 24. Or we could just have the warning and not ignore the value, so it would show in the log as a warning about a possible typo.